Add checklist for creating new standards (#199)
whatwg/meta: NEW-STANDARD.md

When adding a new WHATWG standard, these are the things that need to be done:

1. Create a logo.
1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
   1. Ensure the subdomain works.
   1. Ensure there is repository.
1. Create a Twitter account.
1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)

> 1. Run https://github.com/whatwg/spec-factory.

Reviewer: Why is this necessary? Can it be used to create a new empty repo in a previous step?

Author: It will (should?) fill in some of the boilerplate files.

1. Run `labels.py`.

> 1. Draw the rest of the fucking owl.

Reviewer: I laughed, but I don't know about this :) If one hasn't seen the meme it will seem bizarre and maybe not very welcoming. (There's a chance people looking to get engaged will find their way to this file via a search engine.)

> When adding a new WHATWG standard, these are the things that need to be done:

Reviewer 2: ```suggestion
# New Standard Checklist

When adding a new WHATWG standard, these are the things that need to be done:
```

1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
   1. Ensure the subdomain works and is configured: https://github.com/whatwg/misc-server.

>    1. Ensure there is repository.

Reviewer 2: ```suggestion
   1. Ensure there is a repository under the whatwg GitHub organization.
```

> 1. Create a logo and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.

Reviewer 3: Maybe say the logo should be svg and use the right shade of green?

> 1. Create a logo and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.

Author: ```suggestion
1. Create an SVG logo in the right shade of green (i.e., WHATWG green) and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.
```

1. Create a Twitter account.
   1. Use https://github.com/whatwg/whattweetbot-keys to get the relevant keys. (The Steering Group will help with this.)

> 1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)

Reviewer 2: ```suggestion
1. Add the standard to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
```

> 1. Run https://github.com/whatwg/spec-factory.

Reviewer 2: ```suggestion
1. Run https://github.com/whatwg/spec-factory inside a local checkout of the standard's repository to create the appropriate boilerplate files.
```

although maybe this should be done before creating the repository...

Author: I don't think we support this option currently, but that might be good to add.

Reviewer 2: Oh, I see. I guess leaving as-is then is fine and people will have to read the docs over there to realize that they should run it in the parent directory (?).

Author: I guess I should look at this again (prolly tomorrow) and divide steps a bit between Admin/SG and editor. It's mainly meant as a checklist for Admin/SG, but that is not clear.

> 1. Run `labels.py`.

Reviewer 2: ```suggestion
1. Run [`labels.py`](./labels.py) with appropriate `--repository` and `--token` parameters to create appropriate labels for the new repository.
```

> 1. You should be all set. When in doubt reach out to the Steering Group.

Reviewer 2: ```suggestion

You should be all set. When in doubt reach out to the Steering Group.
```
